packages/my-extended-retail-app/package.json

```diff
@@ -67,9 +67,9 @@
     "scripts": {
         "analyze-build": "cross-env MOBIFY_ANALYZE=true npm run build",
         "build-translations": "npm run extract-default-translations && npm run compile-translations",
-        "compile-translations": "formatjs compile-folder --ast app/translations app/translations/compiled",
-        "compile-translations:pseudo": "formatjs compile --ast app/translations/en-US.json --out-file app/translations/compiled/en-XB.json --pseudo-locale en-XB",
-        "extract-default-translations": "formatjs extract 'app/**/*.{js,jsx}' --out-file app/translations/en-US.json --id-interpolation-pattern [sha512:contenthash:base64:6]",
+        "compile-translations": "formatjs compile-folder --ast pwa-kit-overrides/app/translations pwa-kit-overrides/app/translations/compiled",
+        "compile-translations:pseudo": "formatjs compile --ast pwa-kit-overrides/app/translations/en-US.json --out-file pwa-kit-overrides/app/translations/compiled/en-XB.json --pseudo-locale en-XB",
+        "extract-default-translations": "formatjs extract './node_modules/retail-react-app/app/**/*.{js,jsx}' 'pwa-kit-overrides/app/**/*.{js,jsx}' --out-file pwa-kit-overrides/app/translations/en-US.json --id-interpolation-pattern [sha512:contenthash:base64:6]",
```

Comment:
The `my-extended-retail-app` is going to be deleted. So ultimately this isn't going to be the solution that we are gonna be working with in the end.

Reply:
Can we have the generator to change package.json when a user chooses to use extensibility? 🤔

Reply:
In any way, we still have to edit an override file to get tell react-intl to extract more translations than the base template. It is either in this template, or we override the `fetchTranslation` function.

Reply:
@alexvuong `npm run build-translations` does merge all translations from both base template and the overrides. And this is what Brian wanted. 

I'd imagine that the generated project would have no translation files initially. And that's ok, because they're meant to be generated anyways. Then at some point, `npm run build-translations` would be run and it will extract all the translations by looking into all the js, jsx files in both the base template and overrides dir.